geps/gep-3779/index.md

```diff
@@ -0,0 +1,263 @@
+# GEP-3779: Identity Based Authz for East-West Traffic
+
+* Issue: [#3779](https://github.com/kubernetes-sigs/gateway-api/issues/3779)
+* Status: Provisional
+
+(See [status definitions](../overview.md#gep-states).)
+
+
+## TLDR
+
+Provide a method for configuring Gateway API Mesh implementations to enforce east-west identity-based Authorization controls. At the time of writing this we leave Authentication for specific implementation and outside of this proposal scope.
+
+
+## Goals
+
+(Using the [Gateway API Personas](../../concepts/roles-and-personas.md))
+
+* A way for Ana the Application Developer to configure a Gateway API for Mesh implementation to enforce authorization policy that **allows** or **denies** identity or multiple identities to talk with some set of the workloads she controls.
+
+* A way for Chihiro, the Cluster Admin, to configure a Gateway API for Mesh implementation to enforce non-overridable cluster-wide, authorization policies that **allows** or **denies** identity or multiple identities to talk with some set of the workloads in the cluster.
+
+* A way for both Ana and Chihiro to restrict the scope of the policies they deploy to specific ports.
+
+## Stretch Goals
+
+* A way for Chihiro, the Cluster Admin, to configure a Gateway API for Mesh implementation to enforce default, overridable, cluster-wide, authorization policies that **allows** or **denies** identity or multiple identities to talk with some set of the workloads in the cluster.
```

Comment:
This is in goals and stretch goals I think

Reply:
Ah thanks, I looked at this for a while without noticing that :laughing: 

In that case, I am not sure I understand this... "overridable/default" make sense in contexts where there is 1 policy that wins. Generally authorization policies are ALL enforced, where "any DENY means it is denied, any ALLOW means allowed, else deny".

So what does it mean to have defaults and overrides, etc in this context?

Reply:
@mikemorris Istio used allow-nothing for global deny because of the way envoy filters are placed. But the behavior is same as Consul Global Deny. Users can create finer-grained allow policies to override allow-nothing in Istio. You can compare Istio's allow-nothing same as global deny in Consul. Consul 

But in this proposal we can discuss the UX (API) later but the goal is provide a way to the application user so that can relax the default deny for their workloads.

@LiorLieberman we should think more on the language, override/non-override doesn't go well here imo.
We should talk about precedence instead.

Reply:
Yeah that works, ty!